link_test.go

```diff
@@ -2148,6 +2148,66 @@ func TestLinkSetGROMaxSize(t *testing.T) {
 	}
 }
 
+func TestLinkSetGSOIPv4MaxSize(t *testing.T) {
+	minKernelRequired(t, 6, 3)
+	tearDown := setUpNetlinkTest(t)
+	defer tearDown()
+
+	iface := &Veth{LinkAttrs: LinkAttrs{Name: "foo", TxQLen: testTxQLen, MTU: 1500}, PeerName: "bar"}
+	if err := LinkAdd(iface); err != nil {
+		t.Fatal(err)
+	}
+
+	link, err := LinkByName("foo")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = LinkSetGSOIPv4MaxSize(link, 32768)
```

Comment:
Many times people would learn a functionality by looking at its UT.
Is there any reason for choosing an exact power of 2?
This may mislead people; one would expect 2^N -1, as in networking and computing max sizes are usually because of a N bytes long protocol field/buffer.

Reply:
Power of 2 is okay here. I used the same value as in the existing IPv6 test further above, and the kernel default is also power of 2.